site/source/user-manual/backups/backup-setup/backup-linux.rst

```diff
@@ -0,0 +1,126 @@
+.. _backup-linux:
```

Comment:
In some places, you are using "backup-[platform]" and others you are using "cifs-[platform]". Stick with one convention.